### PR TITLE
src/wordpress/config.py -- Add method comments

### PR DESCRIPTION
**Pull request by LuluTchab**
**From issue**: None

**Low level changes:**

1. Ajout de commentaires pour les méthodes de la classe.

**Targetted version**: N.A

### CODE REVIEW

File: src/wordpress/config.py [hunk 1]
```diff
@@ -65,23 +78,48 @@ def keep_wp_sites(dir_name):
...
+        Argument keywords:
+        command -- WP-CLI command to execute
+        """
         cmd = "wp {} --path='{}'".format(command, self.wp_site.path)
         return Utils.run_command(cmd)
 
     @property
     def is_installed(self):
+        """
+        Check if a WordPress is installed
```
Comment (ebreton) [inline src/wordpress/config.py]: i.e files found

File: src/wordpress/config.py [hunk 2]
```diff
@@ -65,23 +78,48 @@ def keep_wp_sites(dir_name):
...
+        True, False
+        """
         # checkt that index.php has been created
         valid_path = os.path.join(self.wp_site.path, 'index.php')
         return os.path.exists(valid_path)
 
     @property
     def is_config_valid(self):
+        """
+        Tells if WordPress configuration is valid
```
Comment (ebreton) [inline src/wordpress/config.py]: i.e connection to DB successfull with credentials from wp-config.php

File: src/wordpress/config.py [hunk 3]
```diff
@@ -90,9 +128,18 @@ def is_install_valid(self):
...
     @property
     def wp_version(self):
+        """
+        Returns installed WordPress version
+        """
         return self.run_wp_cli('core version')
 
     def config_infos(self, field=None):
+        """
+        Extract and return WordPress configuration informations
```
Comment (ebreton) [inline src/wordpress/config.py]: i.e the ones from `wp-cli config get` (https://developer.wordpress.org/cli/commands/config/get/)

File: src/wordpress/config.py [hunk 4]
```diff
@@ -120,21 +167,39 @@ def config_infos(self, field=None):
...
     @property
     def db_password(self):
+        """
+        Returns WordPress installation DB password for DB_USER
+        """
         return self.config_infos(field='DB_PASSWORD')
 
     def user_infos(self, username=None):
+        """
+        Returns informations about all or one given user.
```
Comment (ebreton) [inline src/wordpress/config.py]: i.e a WPUser instance(s), in a dictionnary where keys are user logins

File: src/wordpress/config.py [hunk 5]
```diff
@@ -165,20 +230,54 @@ def user_infos(self, username=None):
 
     @property
     def admins(self):
+        """
+        Returns a list containing informations about 'administrator' users
```
Comment (ebreton) [inline src/wordpress/config.py]: list of WPUsers

File: src/wordpress/config.py (hunk 1 above)
Comment (LuluTchab) [reply src/wordpress/config.py]: fait

File: src/wordpress/config.py (hunk 2 above)
Comment (LuluTchab) [reply src/wordpress/config.py]: daune

File: src/wordpress/config.py (hunk 3 above)
Comment (LuluTchab) [reply src/wordpress/config.py]: fertig

File: src/wordpress/config.py (hunk 4 above)
Comment (LuluTchab) [reply src/wordpress/config.py]: Gemacht !

File: src/wordpress/config.py (hunk 5 above)
Comment (LuluTchab) [reply src/wordpress/config.py]: done

**Pull request merged**